style.css

```diff
@@ -1 +1,345 @@
 /*  Add your styling here */
+
+body {
+  background-color: #f0d7d2;
+  color: #4285f4;
+  font-family: "Karla", sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 80px;
+}
+
+div h1 {
+  padding-left: 110px;
+}
+
+h1 {
+  font-style: italic;
+  font-weight: lighter;
+  font-family: "Vollkorn", serif;
+  position: relative;
+  color: rgb(189, 150, 52);
+}
+
+.logo {
+  width: 80px;
+  max-width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0px;
+}
+
+.nav-menu {
+  display: block;
+  position: fixed;
+  top: 30px;
+  right: 0px;
+  z-index: 1;
+  user-select: none;
+}
+
+.nav-menu input {
+  display: block;
+  width: 40px;
+  height: 30px;
+  position: absolute;
+  top: -8px;
+  right: 9px;
```

Comment:
This is ok but if your aim is to vertically center something, remember to try using flexbox rules first, before resorting to position: absolute and fixed spacing